pkg/controller/bootstrap/bootstrapsigner.go

```diff
@@ -71,92 +71,103 @@ func DefaultBootstrapSignerOptions() BootstrapSignerOptions {
 
 // BootstrapSigner is a controller that signs a ConfigMap with a set of tokens.
 type BootstrapSigner struct {
-	client          clientset.Interface
-	configMapKey    string
-	secretNamespace string
-
-	configMaps cache.Store
-	secrets    cache.Store
+	client             clientset.Interface
+	configMapName      string
+	configMapNamespace string
+	secretNamespace    string
 
 	// syncQueue handles synchronizing updates to the ConfigMap.  We'll only ever
 	// have one item (Named <ConfigMapName>) in this queue. We are using it
 	// serializes and collapses updates as they can come from both the ConfigMap
 	// and Secrets controllers.
-	syncQueue workqueue.Interface
+	syncQueue workqueue.RateLimitingInterface
+
+	secretLister    corelisters.SecretLister
+	configMapLister corelisters.ConfigMapLister
```

Comment:
My personal preference is to group the lister & synced vars together, i.e.
```go
secretLister    corelisters.SecretLister
secretSynced    cache.InformerSynced
```

Reply:
Updated